src/content/en/fundamentals/getting-started/service-worker/service-worker-install-step.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: shared/narrow
+title: "Service Worker Install Step"
+description: "After a controlled page kicks off the registration process, let's shift to the point of view of the service worker script, which handles the `install` event."
+published_on: 2014-12-01
+updated_on: 2016-01-19
+translation_priority: 0
+order: 6
+authors:
+  - mattgaunt
+---
+
+After a controlled page kicks off the registration process, let's shift to the
+point of view of the service worker script, which handles the `install` event.
+
+For the most basic example, you need to define a callback for the install event
+and decide which files you want to cache.
+
+{% highlight javascript %}
+// The files we want to cache
```

Comment:
Cut these lines, you only really need to show the eventListener here, the other stuff isn't important... yet.